marginalia.el

```diff
@@ -234,6 +234,38 @@ determine it."
   '((t :inherit font-lock-preprocessor-face))
   "Face used to highlight file owners in `marginalia-mode'.")
 
+(defface marginalia-file-priv-no
+  '((t :inherit shadow))
+  "Face used to highlight the no privilege attribute in `marginalia-mode'.")
+
+(defface marginalia-file-priv-dir
+  '((t :foreground "#51afef"))
```

Comment:
Please change the color to font-lock colors, just inherit something which makes sense.

Reply:
It won't look as good, but I'll try to work out a somewhat sensible scheme.

Reply:
Done :)